save-demo/src/main/kotlin/com/saveourtool/save/demo/entity/GithubRepo.kt

```diff
@@ -15,4 +15,9 @@ class GithubRepo(
     var organizationName: String,
     @Column(name = "project_name")
     var toolName: String,
-) : BaseEntity()
+) : BaseEntity() {
+    /**
+     * @return pretty string that defines [GithubRepo]
+     */
+    fun toPrettyString() = "$organizationName/$toolName"
```

Comment:
btw, why it's called github repo, while I can't see anything corresponding to `github`?

Reply:
The point it that the repo that makes sense only in terms of GitHub (https://github.com/saveourtool/save-cloud/pull/1637#discussion_r1052927584)